database/Author.js

```diff
@@ -0,0 +1,78 @@
+ import db from './db'
+
+ const addAuthor = `INSERT INTO author (name) VALUES ($1) RETURNING *`
+ const getOneAuthor = `SELECT * FROM author WHERE id=$1`
+ const getAllAuthors = `SELECT * from author`
+ const editAuthor = `UPDATE author SET name=$1 WHERE id=$2 RETURNING *`
+ const deleteAuthor = `DELETE FROM author where id=$1`
+
+const getByName = 'SELECT * FROM author WHERE name=$1'
+
+const Author = {
+
+    add: ( request, response, next ) => {
+
+      let { name } = request.body
+      db.query(addAuthor, name)
+        .then( author => response.status(200).json({
+          status: 'Success',
+          data: author,
+          message: 'Inserted author into database.'
+        }))
+        .catch( error => next( error ) )
+      },
+
+    getOne: ( request, response, next ) => {
+      let { id } = request.params
+      db.query(getOneAuthor, id)
+        .then ( author => response.status(200).json({
+          status: 'Success',
+          data: author,
+          message: 'Retrieved author.'
+        }))
+        .catch( error => next( error ) )
+    },
+
+    getAll: (request, response, next ) => {
+      db.query(getAllAuthors)
+        .then( authors => response.status(200).json({
+          status: 'Success',
+          data: authors,
+          message: 'Retrieved all authors.'
+        }))
+        .catch( error => next( error ) )
+    },
+
+    edit: ( request, response, next ) => {
+      let { id, name } = request.body
+      db.query(editAuthor, [name, id])
+        .then( author => response.status(200).json({
+          status: 'Success.',
+          data: author,
+          message: 'Updated author.'
+        }))
+        .catch( error => next( error ) )
+    },
+
+    delete: ( request, response, next ) => {
+      let { id } = request.params
+      db.query(deleteAuthor, id)
+      .then( response.status(200).json({
+      status: 'Success',
+      message: 'Deleted author from database.'
+      }))
+      .catch( error => next( error ) )
+    },
+
+    noAPI: {
+      getByName: name => {
+        return db.query( getByName, name )
+        .then( author => author )
+      },
+       add: name => {
+        return db.query( addAuthor, name )
```

Comment:
Stylistically, I prefer being explicit. In this particular case I think that db.query is a little bit more ambiguous than using pg-promise's none(), one(), or any() methods since your queries are pretty straight forward and you aren't using the QRM (qrm - (optional) Query Result Mask, as explained below. When not passed, it defaults to pgp.queryResult.any.) 
So this might be the reason you have to index into an array rather just getting back one row if you were to use db.one()